src/main/java/ladder/domain/Point.java

```diff
@@ -0,0 +1,55 @@
+package ladder.domain;
+
+import java.util.Objects;
+import java.util.Random;
+
+public class Point {
+    public static final String LINE_STATE_FALSE = "     |";
+    public static final String LINE_STATE_TRUE = "-----|";
+
+    private final boolean nowPoint;
+
+    public Point() {
+        this(true);
+    }
+
+    public Point(boolean beforePoint) {
+        this.nowPoint = setNextState(beforePoint);
+    }
+
+    public static boolean setNextState(boolean beforePoint) {
+        if (beforePoint) {
+            return false;
+        }
+        return new Random().nextBoolean();
+    }
+
+    private static String getStateShape(boolean lineState) {
+        if (lineState) {
+            return LINE_STATE_TRUE;
+        }
+        return LINE_STATE_FALSE;
+    }
+
+    public boolean getNowPoint() {
```

Comment:
```suggestion
    public boolean isNowPoint() {
```
윈도우도 단축키가 동일한지는 모르겠네요..
아마 alt + insert 키를 누르면 Generate이라는 메뉴가 뜰텐데 그중에 getter가 있을거에요
선택하고 nowPoint:boolean을 선택, ok 누르시면 제가 제안한 코드와 동일한 모양으로 나올거에요
보통 boolean 변수의 getter는 is를 붙여줍니다.

Reply:
반영했습니다!